src/components/SQForm/SQFormMultiSelect.js

```diff
@@ -92,6 +92,19 @@ function SQFormMultiSelect({
     setToolTipEnabled(!toolTipEnabled);
   };
 
+  /**
+   * this handles scenarios where label and value are not the same,
+   * e.g., if value is an "ID"
+   */
+  const getRenderValue = selected => {
+    const selectedChildren = children
+      ?.filter(child => selected.includes(child.value))
+      ?.map(child => child.label)
+      ?.join(', ');
```

Comment:
`.reduce()` would only require 1 iteration of all items instead of 2. I'm fine considering this a premature optimization if we didn't notice any performance issues.